README.md

```
@@ -71,7 +75,7 @@ Check out our current openings and apply via [Stream's website](https://getstrea
   </tr>
   <tr>
     <td> A list of channels matching provided query </td>
-    <th rowspan="9"><img src="ReadmeAssets/ChannelListPreview.gif?raw=true" width="80%" /></th>
+    <th rowspan="9"><img src="ReadmeAssets/Channel_List_Bezel.png" width="80%" /></th>
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)